<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileConfigBase</class>
 <widget class="QWidget" name="FileConfigBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="lblPath">
     <property name="text">
      <string>Path for incoming files:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EditFile" name="edtPath" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="grpAccept">
     <property name="title">
      <string>Accept mode</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="btnDialog">
        <property name="text">
         <string>Show response</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="btnAccept">
        <property name="text">
         <string>Auto accept</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="chkOverwrite">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Overwrite files</string>
        </property>
        <property name="buttonGroupId" stdset="0">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="btnDecline">
        <property name="text">
         <string>Auto decline</string>
        </property>
        <property name="buttonGroupId" stdset="0">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPlainTextEdit" name="edtDecline">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditFile</class>
   <extends>QWidget</extends>
   <header>simgui/editfile.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnAccept</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkOverwrite</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDecline</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtDecline</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
